### PR TITLE
Create ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by tba

Ticket: [MTM-59023](https://cumulocity.atlassian.net/browse/MTM-59023)
Original PR: [6207](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6207)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: load dynamic options from app specific path (#6207)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Support loading dynamic options from app-specific paths
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59023
+version: 1020.0.0
+---
+load dynamic options from app specific path (#6207)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility for configuring applications, Cumulocity IoT now supports loading dynamic options from application-specific paths. With this change, each hosted application can define its own set of dynamic options in a file located at a designated path within the application directory. This allows application-specific configurations to be loaded dynamically, enabling easier customization and management of application settings without modifying the global platform configuration.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md (hunk 2 above)
Comment (reey) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md:17]:
```suggestion
To provide more flexibility for configuring applications, Cumulocity IoT now supports loading dynamic options from application-specific paths. With this change, each hosted application will load its dynamic options from a path that includes the applications context path. This allows application-specific configurations to be loaded dynamically.
The Cockpit application would now load it's dynamic options from: `/apps/public/public-options@app-cockpit/options.json` while it previously did so from: `/apps/public/public-options/options.json`.
This change was done as a preparation for upcoming changes to the branding manager.
```

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59023
+version: 1020.0.0
+---
+To provide more flexibility for configuring applications, Cumulocity IoT now supports loading dynamic options from application-specific paths. With this change, each hosted application will load its dynamic options from a path that includes the applications context path. This allows application-specific configurations to be loaded dynamically.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md:17]:
```suggestion
To provide more flexibility for configuring applications, Cumulocity IoT now supports loading dynamic options from application-specific paths. With this change, each hosted application will load its dynamic options from a path that includes the application context path. This allows application-specific configurations to be loaded dynamically.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: Support loading dynamic options from app-specific paths
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-load-dynamic-options-from-app-specific-path-6207.md:3]:
```suggestion
title: Support for loading dynamic options from app-specific paths
```

**Review by rahultsag — APPROVED** [review]

**Pull request merged**